Dockerfile

```diff
@@ -1,22 +1,19 @@
-FROM ghcr.io/imagegenius/baseimage-alpine:3.20
+FROM ghcr.io/linuxserver/baseimage-alpine:3.20
 
 # set version label
 LABEL maintainer="ozpinbeacon"
 
 RUN \
   echo "**** install runtime packages ****" && \
   apk add --no-cache \
-	python3 \
-		py3-flask \
-		py3-sqlalchemy \
-		py3-apscheduler \
-		py3-ldap3 
+    py3-apscheduler \
+    py3-flask \
+    py3-ldap3 \
+    py3-sqlalchemy \
+    python3
 
 # copy local files
 COPY root/ /
 
-# Execute the service
-CMD ["flask", "--app", "ADOverseas", "run", "--host", "0.0.0.0", "--port", "3500"]
```

Comment:
`CMD` is only needed when we are not using a supervisor, the baseimage here has [s6-overlay](https://github.com/just-containers/s6-overlay) controlled in `etc/s6-rc.d`